src/style.css

```diff
@@ -0,0 +1,28 @@
+body,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.list {
+  display: flex;
+}
+
+.list__item {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  list-style: none;
+  margin-right: 4px;
```

Comment:
Margin-right should not be added for the last element.

Reply:
> Margin-right should not be added for the last element.

Dobrze, poprawie to

Reply:
> Margin-right should not be added for the last element.

Fixed